backend/core/package.json

```diff
@@ -19,26 +19,25 @@
     "test-circular": "yarn depcruise --validate .dependency-cruiser.js .",
     "test-linter": "yarn linter-base-config .",
     "test-type": "tsc --noEmit",
-    "test-unit": "jest --runInBand --collectCoverage --logHeapUsage --passWithNoTests"
+    "test-unit": "vitest run --coverage --passWithNoTests"
   },
   "dependencies": {
     "@swarmion-starter/serverless-configuration": "1.0.0",
     "@swarmion/serverless-helpers": "0.8.3"
   },
   "devDependencies": {
     "@serverless/typescript": "^3.19.0",
-    "@swarmion-starter/configuration": "1.0.0",
     "@swarmion-starter/core-contracts": "1.0.0",
-    "@types/jest": "^27.5.2",
     "@types/node": "^17.0.45",
+    "c8": "^7.12.0",
     "dependency-cruiser": "^11.10.0",
     "esbuild": "^0.14.47",
     "eslint": "^8.18.0",
-    "jest": "^27.5.1",
     "serverless": "^3.19.0",
     "serverless-esbuild": "^1.30.0",
-    "ts-jest": "^27.1.5",
     "ts-node": "^10.8.1",
-    "typescript": "^4.7.4"
+    "typescript": "^4.7.4",
+    "vite": "^3.0.3",
+    "vitest": "^0.19.1"
```

Comment:
you don't need the `vite` explicit dependency. You don't need to use `vite` in a backend service